designer/server/src/main/scala/pl/touk/nussknacker/ui/api/SettingsResources.scala

```diff
@@ -95,7 +98,8 @@ object TopTabType extends Enumeration {
                                             tabs: Option[List[TopTab]],
                                             intervalTimeSettings: IntervalTimeSettings,
                                             testDataSettings: TestDataSettings,
-                                            redirectAfterArchive: Boolean
+                                            redirectAfterArchive: Boolean,
+                                            usageStatisticsReports: UsageStatisticsReportsSettings,
```

Comment:
Do you know what is the rule for choosing whether we should add sth directly to `UISettings` or to `ToggleFeaturesOptions`. `AnalyticsConfig` we have on top level.

Reply:
Unfortunetly I don't know.. maybe `analytics: Option[AnalyticsSettings]` should be moved to `ToggleFeaturesOptions` ?

Reply:
but it's not relatetd to this change, so can you e.g. create issue for later and resolve this comment?

Reply:
Or everything should be flatten :D Let's leave it as it is ;)